lib/run_keeper/activity_request.rb

```diff
@@ -15,24 +15,16 @@ def limit_results activities
 
     def request activities = nil
       response   = @runkeeper.request('fitness_activities', @options[:params])
-      activities = response.parsed['items'].map { |activity| Activity.new(activity) }
+      activities = response.parsed['items'] ? response.parsed['items'].map { |activity| Activity.new(activity) } : []
```

Comment:
I'd probably have done `Array(response.parsed['items']).map { .... }` instead of a ternary but this will work.